Framework/Algorithms/inc/MantidAlgorithms/CalculateCarpenterSampleCorrection.h

```diff
@@ -0,0 +1,98 @@
+#ifndef MANTID_ALGORITHM_MULTIPLE_SCATTERING_ABSORPTION_H_
+#define MANTID_ALGORITHM_MULTIPLE_SCATTERING_ABSORPTION_H_
+
+#include "MantidAPI/AlgorithmManager.h"
```

Comment:
Do we need most of these headers here? At a glance I think we only need `#include "MantidAPI/DataProcessorAlgorithm.h`, `MantidAPI/MatrixWorkspace.h`,`#include "MantidHistogramData/Points.h` and `<vector>`?